x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -6,59 +6,67 @@
  */
 
 import {
-  EuiInMemoryTable,
   EuiBasicTableColumn,
   EuiLink,
   EuiTitle,
   EuiFlexGroup,
   EuiFlexItem,
   EuiSpacer,
   EuiText,
+  EuiIconTip,
 } from '@elastic/eui';
 import React from 'react';
-import styled from 'styled-components';
 
-import { SummaryRow } from './helpers';
+import type { AlertSummaryRow } from './helpers';
+import * as i18n from './translations';
 import { VIEW_ALL_FIELDS } from './translations';
+import { SummaryTable } from './table/summary_table';
+import { SummaryValueCell } from './table/summary_value_cell';
+import { PrevalenceCellRenderer } from './table/prevalence_cell';
 
-export const Indent = styled.div`
-  padding: 0 12px;
-`;
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export const StyledEuiInMemoryTable = styled(EuiInMemoryTable as any)`
-  .euiTableHeaderCell,
-  .euiTableRowCell {
-    border: none;
-  }
-  .euiTableHeaderCell .euiTableCellContent {
-    padding: 0;
-  }
-
-  .flyoutOverviewDescription {
-    .hoverActions-active {
-      .timelines__hoverActionButton,
-      .securitySolution__hoverActionButton {
-        opacity: 1;
-      }
-    }
+const summaryColumns: Array<EuiBasicTableColumn<AlertSummaryRow>> = [
+  {
+    field: 'title',
+    truncateText: false,
+    name: i18n.HIGHLIGHTED_FIELDS_FIELD,
+    textOnly: true,
+  },
+  {
+    field: 'description',
+    truncateText: false,
+    render: SummaryValueCell,
+    name: i18n.HIGHLIGHTED_FIELDS_VALUE,
+  },
+  {
+    field: 'description',
+    truncateText: true,
+    render: PrevalenceCellRenderer,
+    name: (
+      <>
+        {i18n.HIGHLIGHTED_FIELDS_ALERT_PREVALENCE}{' '}
+        <EuiIconTip
+          type="iInCircle"
+          color="subdued"
+          title="Alert Prevalence"
+          content={<span>{i18n.HIGHLIGHTED_FIELDS_ALERT_PREVALENCE_TOOLTIP}</span>}
+        />
+      </>
+    ),
+    align: 'right',
+    width: '130px',
+  },
+];
 
-    &:hover {
-      .timelines__hoverActionButton,
-      .securitySolution__hoverActionButton {
-        opacity: 1;
-      }
-    }
-  }
-`;
+const rowProps = {
+  // Class name for each row. On hover of a row, all actions for that row will be shown.
+  className: 'flyoutTableHoverActions',
+};
 
-export const SummaryViewComponent: React.FC<{
+const SummaryViewComponent: React.FC<{
```

Comment:
Please, add a unit tests for this component

Reply:
Missed this one, thanks! I've added it!